.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+# Auto formats python code
+repos:
+-    repo: https://github.com/psf/black.git
+     rev: 23.1a1
+     hooks: 
+     - id: black
+          name: Black Python code formatting
+          entry: bash -c 'black "$@"; git add -u' --
+          language: python
+          types: [python]
+          args: ["--line-length=120"]
```

Comment:
Do you actually need all these args or will it run with just the `id: black` arg?

Reply:
It did run with just `id:black` locally. I guess the rest can be removed.